tests/storage/snapshot.go

```diff
@@ -1470,6 +1473,70 @@ var _ = SIGDescribe("VirtualMachineSnapshot Tests", func() {
 				Expect(snapshot.Status.SnapshotVolumes.ExcludedVolumes).Should(HaveLen(1))
 				Expect(snapshot.Status.SnapshotVolumes.ExcludedVolumes[0]).Should(Equal("notsnapshotablevolume"))
 			})
+
+			It("[Serial]Should also include backend PVC in the snapshot", Serial, func() {
```

Comment:
You can drop the [Serial] string, label is enough

Reply:
ack